lib/src/options.dart

```diff
@@ -46,7 +46,7 @@ class I18NextOptions with Diagnosticable {
   /// current namespace.
   ///
   /// Defaults to null.
-  final String? fallbackNamespace;
+  final List<String>? fallbackNamespaces;
```

Comment:
Could you update the comment here as well?

Reply:
fixed